master/buildbot/db/model.py

```diff
@@ -314,7 +314,7 @@ class Model(base.DBConnectorComponent):
                   nullable=False),
         sa.Column('property_name', sa.String(256), nullable=False),
         # JSON-encoded tuple of (value, source)
-        sa.Column('property_value', sa.String(1024), nullable=False),
+        sa.Column('property_value', sa.String(14336), nullable=False),
```

Comment:
I dont think it is a good idea. We should keep Text, and be coherent with the other property table.